backend/src/controllers/mural.controller.ts

```diff
@@ -6,22 +6,49 @@ import { MuralService } from "../services/mural.service";
 export class MuralController {
   public muralService: MuralService = new MuralService();
 
-  // GET all /mural
+  /**
+   * GET /mural to get all murals (paginated)
+   * @param req HTTP request optionally containing page count and size in querystring:
+   * limit: number (default 40)
+   * page: number (default 0)
+   * @param res HTTP response containing one page worth of mural data
+   */
   public async showAll(req: Request, res: Response) {
-    const limit = Number(req.query.limit ?? 40);
-    const offset = Number(req.query.page ?? 0) * limit;
     try {
+      const limit = Number(req.query.limit ?? 40);
+      const offset = Number(req.query.page ?? 0) * limit;
       const murals = await this.muralService.showAll(limit, offset);
       res.status(202).json(murals);
     } catch (e) {
       res.status(500).json({ error: "Something went wrong." });
     }
   }
 
-  // POST /mural
+  /**
+   * POST /mural to create a new mural based on MuralInterface parameters
+   * @param req HTTP request with body:
+   *  {
+      name: string;
+      artistId: number;
+      year: number;
+      address: string;
+      city: string;
+      boroughId: number;
+      neighbourhood?: string;
+      description?: string;
+      partners?: string[];
+      assistants?: string[];
+      longitude: number;
+      latitude: number;
+      imgURLs?: string[];
+      socialMediaURLs?: string[];
+    }
+   * @param res HTTP response containing data of the created mural
+   */
   public async create(req: Request, res: Response) {
     const params: MuralInterface = req.body;
     try {
+      //TODO: potentially validate if longitude and latitude are valid in Montreal?
```

Comment:
the mapbox geocoding API should be restricting our search to Montreal but then again our API is just open right now